lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlockBlobFileSystemE2E.cs

```diff
@@ -35,7 +36,9 @@ namespace Org.Apache.REEF.IO.Tests
     /// </summary>
     public sealed class TestAzureBlockBlobFileSystemE2E : IDisposable
     {
-        private const string SkipMessage = "Fill in credentials before running test"; // Use null to run tests
+        // Uncomment SkipMessage = null to run tests
+        private const string SkipMessage = "Fill in credentials before running test";
+        // private const string SkipMessage = null;
         private const string HelloFile = "hello";
         private IFileSystem _fileSystem;
```

Comment:
all member variables here can be `readonly`

Reply:
fixed

---
In reply to: [198679115](https://github.com/apache/reef/pull/1473#discussion_r198679115) [](ancestors = 198679115)